### PR TITLE
Instamojo - Payment Processor Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://futuregenapps.com/
https://www.fivemorselsoflove.in/
https://www.guliataxi.com/
https://7host.in/#pricing
https://docs.instamojo.com/reference/payments-api

image

image attached for the reference, how **Instamojo** Integration happens in the Websites

### CODE REVIEW

File: src/technologies/i.json [hunk 1]
```diff
@@ -712,6 +712,16 @@
     "scriptSrc": "instafeed\\.nfcube\\.com/",
     "website": "https://apps.shopify.com/instafeed"
   },
+  "Instamojo": {
+    "cats": [
+      41
+    ],
+    "description": "Best Payment Gateway Integration in PHP, JAVA, Python, Wordpress SDK & API, Web, Android - Instamojo",
+    "dom": "a[href*='instamojo\\.com'], a[rel*='im-checkout']",
```
Comment (nurbek91) [inline src/technologies/i.json:720]: ` "dom": "a[href*='.instamojo\\.com/']",`

File: src/technologies/i.json [hunk 2]
```diff
@@ -712,6 +712,16 @@
...
+    "icon": "instamojo.png",
+    "scriptSrc": "//im\\-embed/",
```
[the remark this answers is not in the document]
Comment (nurbek91) [reply src/technologies/i.json:722]: for checkout page https://www.instamojo.com/@futuregenapps
```
 "js": {
      "INITIAL_STATE.seller.avatar": "\\.instamojo\\.com/"
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR. Need improvements

File: src/technologies/i.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (nurbek91) [reply src/technologies/i.json]: or try to find examples and share, I'll see what can be done.
If there are no examples, it is better to delete

File: src/technologies/i.json [hunk 3]
```diff
@@ -712,6 +712,19 @@
     "scriptSrc": "instafeed\\.nfcube\\.com/",
     "website": "https://apps.shopify.com/instafeed"
   },
+  "Instamojo": {
+    "cats": [
+      41
+    ],
+    "description": "Instamojo is a Bangalore-based company that is targeted to selling digital goods and collecting payment online.",
+    "dom": "a[href*='.instamojo\\.com/'], a[rel*='im-checkout']",
```
Comment (nurbek91) [inline src/technologies/i.json:720]: a[rel*='im-checkout']" 
this part can also be removed. I don't seem to notice any traces

File: src/technologies/i.json [hunk 4]
```diff
@@ -712,6 +712,19 @@
...
+    "icon": "instamojo.svg",
+    "js": {
+      "INITIAL_STATE.seller.avatar": "\\.instamojo\\.com/"
+    },
+    "scriptSrc": "//im\\-embed/im\\-embed\\.min\\.js",
```
Comment (nurbek91) [inline src/technologies/i.json:725]: or try to find examples and share, I'll see what can be done.
If there are no examples, it is better to delete

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/i.json (hunk 3 above)
Comment (dhavalveera) [reply src/technologies/i.json:720]: can you please review the screenshot I've shared from the Instamojo API Docs, from that I've added this trace, if you think it's not required, then I can remove it

File: src/technologies/i.json (hunk 4 above)
Comment (dhavalveera) [reply src/technologies/i.json:725]: can you please review the screenshot I've shared from the Instamojo API Docs, from that I've added this trace, if you think it's not required, then I can remove it

Comment (nurbek91) [reply src/technologies/i.json:725]: Yes, I looked, I need examples
the current option does not look safe and there will be many false positive results

Comment (dhavalveera) [reply src/technologies/i.json]: done

Comment (dhavalveera) [reply src/technologies/i.json]: removed the both traces which you mentioned.

File: src/technologies/i.json [hunk 5]
```diff
@@ -712,6 +712,18 @@
...
+  "Instamojo": {
+    "cats": [
+      41
+    ],
+    "description": "Instamojo is a Bangalore-based company that is targeted to selling digital goods and collecting payment online.",
+    "dom": "a[href*='.instamojo\\.com/']",
+    "icon": "instamojo.svg",
+    "js": {
+      "INITIAL_STATE.seller.avatar": "\\.instamojo\\.com/"
+    },
```
Comment (nurbek91) [inline src/technologies/i.json:724]: I found examples where scriptSrc is used, we can determine them by js object
https://ugcportal.com/
https://thearchitectsdiary.com/

```
"js": {
      "INITIAL_STATE.seller.avatar": "\\.instamojo\\.com/",
      "Instamojo": ""
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/i.json (hunk 5 above)
Comment (dhavalveera) [reply src/technologies/i.json]: added.

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/i.json (hunk 5 above)
Comment (dhavalveera) [reply src/technologies/i.json]: I checked the both examples you shared, but when I'm checking the page source, I'm not able to find the `scriptSrc` part for the trace @nurbek91

Comment (nurbek91) [reply src/technologies/i.json]: > I checked the both examples you shared, but when I'm checking the page source, I'm not able to find the `scriptSrc` part for the trace @nurbek91

image

**Pull request merged**